examples/shopping_ads/add_performance_max_retail_campaign.py

```diff
@@ -439,6 +446,22 @@ def _create_asset_group_operation(
     )
     operations.append(mutate_operation)
 
+    # Create a ListingGroupFilter and link it to the AssetGroup.
+    # This is necessary for the campaign to run.
+    # https://developers.google.com/google-ads/api/reference/rpc/v9/AssetGroupListingGroupFilter
+    mutate_operation = client.get_type("MutateOperation")
+    asset_group_listing_group = mutate_operation.asset_group_listing_group_filter_operation.create
+    asset_group_listing_group.asset_group = asset_group_service.asset_group_path(
+            customer_id,
+            _ASSET_GROUP_TEMPORARY_ID,
+        )
+    asset_group_listing_group.type_ = client.enums.ListingGroupFilterTypeEnum.UNIT_INCLUDED
+    # Here are other options for ListingGroupFilterTypeEnum
+    # https://developers.google.com/google-ads/api/reference/rpc/v9/ListingGroupFilterTypeEnum.ListingGroupFilterType
```

Comment:
Can you replace the above comments with the below:

```
# Because this is a Performance Max campaign for retail, we need to specify that this is in the shopping vertical.
```

Reply:
will do!